pynest/nest/lib/hl_api_types.py

```diff
@@ -737,16 +737,15 @@ def get(self, keys=None, output=''):
         if pandas_output and not HAVE_PANDAS:
             raise ImportError('Pandas could not be imported')
 
-        # Return empty tuple if we have no connections or if we have done a
-        # nest.ResetKernel()
+        # Return empty tuple if we have no connections or if we have done a nest.ResetKernel()
         num_conn = GetKernelStatus('num_connections')
         if self.__len__() == 0 or num_conn == 0:
             return ()
 
         if keys is None:
             cmd = 'GetStatus'
         elif is_literal(keys):
-            cmd = 'GetStatus {{ /{0} get }} Map'.format(keys)
+            cmd = 'GetStatus'
```

Comment:
sorry, I don't understand why this line changed

Reply:
When we have a `SynapseCollection` with 2 or more synapses, for instance `static_synapse` and `stdp_synapse`, and ask for a parameter that is only part of one of them, like `alpha`, this will throw an error, because alpha is not part of the `static_synapse` dictionary. So now we get all dictionaries and look for the parameter on PyNEST level.

Reply:
ok, of course, it goes with the other change!
maybe a comment here would be useful though, just saying that it will be done in ``restructure_data``?

Reply:
Done!